CMakeLists.txt

```diff
@@ -47,6 +47,14 @@ set(H3_SOVERSION 1)
 
 project(h3 VERSION ${H3_VERSION})
 
+set(H3_FLAGS "")
```

Comment:
Where are the other flags? I'd remove them from `cmake/toolchain.cmake` and put them here instead.

Reply:
Seems reasonable. I've moved the other flags here.